scitos_drivers/package.xml

```diff
@@ -6,13 +6,13 @@
 	Driver for Scitos-G5 using the MIRA framework. Includes drivers for other typical Scitos Hardware, eg laser scanner.
   </description>
 
-  <maintainer email="cburbridge@gmail.com">Chris Burbridge</maintainer>
+  <maintainer email="marc@hanheide.net">Marc Hanheide</maintainer>
 
   <license>GPL</license>
 
   <url type="website">http://github.com/strands-project/scitos_drivers</url>
 
-  <author email="cburbridge@gmail.com">Chris Burbridge</author>
+  <author email="marc@hanheide.net">Marc Hanheide</author>
```

Comment:
I agree that unfortunately I can't maintain this any more, but I think I should still be listed as author, instead just add another author tag?